Assets/Scripts/Util/SpriteUtil.cs

```diff
@@ -8,7 +8,7 @@
 using System.Text.RegularExpressions;
 using UnityEngine;
 
-public static class SpriteUtil 
+public static class SpriteUtil
 {
     const string FILE_PATH = "Sprites/"; // Path within Resources Folder
```

Comment:
Should be made a child of ResoureLoader to reduce the number of places we're redeclaring constants. Also realize this somewhat conflicts with the class I wrote in the scouting PR: SpritesheetSystem. Think we should work to merge them together (and likely better they not be a static class for OOP cohesion ).